src/_scss/pages/modals/settings/_settingsModal.scss

```diff
@@ -39,13 +39,73 @@
         padding: rem(12);
         padding-top: rem(32);
         width: rem(240);
+        overflow: hidden;
 
         h1 {
             margin: 0;
             margin-bottom: rem(25);
             font-size: rem(25);
             font-weight: bold;
         }
+
+        h2 {
+            margin-bottom: 0;
+            margin-top: rem(45);
+            font-size: rem(16);
+            font-weight: bold;
+
+            svg {
+                height: rem(16);
+                width: rem(16);
+                margin-right: rem(5);
+            }
+        }
+
+        .settings-agency-filter {
+            @import '../../dashboard/filters/selectedFilters';
+            @import '../../../components/typeahead/typeahead';
+
+            .typeahead-holder {
+                position: relative;
+        
+                input {
+                    color: #adadad;
```

Comment:
Looks close to $color-gray-light to reuse a variable.

Reply:
This is a direct copy-paste of what we have in the current agency filter for the dashboard in terms of color. I wanted to keep them the same.